### DISCUSSION TITLE
How to remove "Share On" buttons from page

### DISCUSSION THREAD

**Discussion by wkeenan21** [academicpages/academicpages.github.io#2682] [Q&A] [answered]
These buttons: 
![image](https://github.com/user-attachments/assets/59275ccd-d1a4-4c68-a14e-7b9bb204ce35)
appear at the bottom of the page in several places. Is there a quick and easy way to remove them? I found the file that creates the HTML for this page (_includes/social-share.html) but I don't know how to turn it off.

**Comment by rjzupkoii** [comment, marked as answer]
If you don't have any of those set in your `_config.yml` then they will not appear at all. However, do you want to have those set, but just now show in the footer? That's not currently a configuration option, but easy an easy enough one to add if you want to add it to the [Issues](https://github.com/academicpages/academicpages.github.io/issues)

**Reply by wkeenan21** [reply]
Ah I see. share: false in the Defualts section of the config turns this off. That's what I was lookin for.
